bedrock/firefox/templates/firefox/hello/index.html

```diff
@@ -25,15 +27,28 @@
 {% block site_header %}
   <header id="masthead">
     <div class="container">
-      <a href="{{ url('mozorg.home') }}" id="tabzilla" data-infobar="{{ settings.TABZILLA_INFOBAR_OPTIONS }}">Mozilla</a>
+      <div id="hello-header-nav">
+        <a href="{{ url('mozorg.home') }}" id="tabzilla" data-infobar="{{ settings.TABZILLA_INFOBAR_OPTIONS }}">Mozilla</a>
 
-      {% block site_header_nav %}
-        {% include 'firefox/includes/top-menu.html' %}
-      {% endblock %}
+        {% block site_header_nav %}
+          {% include 'firefox/includes/top-menu.html' %}
+        {% endblock %}
+      </div>
 
       {% block site_header_logo %}
-        {{ super() }}
-      {% endblock %}
+        <div id="hello-logo-wrapper">
+          {{ super() }}
+
+          {%
+          set share_urls = {
+            'twitter': 'http://mzl.la/1wz83AN',
+            'googleplus': 'https://www.mozilla.org/' + LANG + '/firefox/hello/?utm_source=Google&utm_medium=social&utm_campaign=Hello&utm_content=WebsiteShare',
```

Comment:
As Twitter uses a short URL above, I'm assuming we're sharing a non-locale specific URL there - but here a locale specific one? Should they be the same?